.github/workflows/ci.yml

```diff
@@ -112,16 +118,18 @@ jobs:
         run: make test_fluvio_install
 
       - name: Install Fluvio CDK
-        run: fluvio install cdk
+        run: fluvio install cdk --develop
 
       - name: Install Rust ${{ matrix.rust }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
-          target: wasm32-unknown-unknown
+          target:
+            - wasm32-unknown-unknown
```

Comment:
why do we need this target?

Reply:
Actually I think we dont, I could remove it of course!